src/Microsoft.ServiceFabric.Services.Remoting/V2/Runtime/IExceptionSerializerSettings.cs

```diff
@@ -0,0 +1,12 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Services.Remoting.V2.Runtime
+{
+    internal interface IExceptionSerializerSettings
```

Comment:
Just a nit for the future PRs: types have internal visibility by default, so I think it's better to not specify it explicitly. It's similar to how the interface members have default visibility that cannot be specified at all.